CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [Version 1.3.0](https://github.com/dataiku/dss-plugin-nlp-preparation/releases/tag/v1.2.2) - New feature - 2023-04
```

Comment:
```suggestion
## [Version 1.3.0](https://github.com/dataiku/dss-plugin-nlp-preparation/releases/tag/v1.3.0) - New feature - 2023-04
```

Hiya! I was looking at your PR to see how to do it for mine, and I noticed you missed the version number in the URL 🙂